webforces/server/mongodbworker.py

```diff
@@ -1,3 +1,6 @@
+import sys
+sys.path.append('../')
+sys.path.append('../../')
```

Comment:
Desired working directory for django is root directory. If you wanna create application for testing db maybe it makes sense to create separate .py file in the root. What do you think about it?

Reply:
It is a temporary solution for debugging.